testing/TestHarness/TestHarness.Mobile/TestHarness.Mobile.csproj

```diff
@@ -1,22 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
 		<TargetFrameworks Condition="'$(UnoTargetFrameworkMobileOverride)'!=''">$(UnoTargetFrameworkMobileOverride)</TargetFrameworks>
-		<TargetFrameworks Condition="'$(UnoTargetFrameworkMobileOverride)'==''">net7.0-android;net7.0-ios;net7.0-maccatalyst;net7.0-macos</TargetFrameworks>
+		<TargetFrameworks Condition="'$(UnoTargetFrameworkMobileOverride)'==''">net8.0-android;net8.0-ios;net8.0-maccatalyst;net8.0-macos</TargetFrameworks>
 		<SingleProject>true</SingleProject>
 		<OutputType>Exe</OutputType>
 		<AssemblyName>TestHarnessApp</AssemblyName>
-		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net7.0-ios'">iossimulator-x64</RuntimeIdentifier>
-		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net7.0-maccatalyst'">maccatalyst-x64</RuntimeIdentifier>
-		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net7.0-macos'">osx-x64</RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net8.0-ios'">iossimulator-x64</RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net8.0-maccatalyst'">maccatalyst-x64</RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net8.0-macos'">osx-x64</RuntimeIdentifier>
```

Comment:
these conditions should be refactored to match what we use in the template which does not care what version of .NET we use

Reply:
Yeh I was thinking of doing that and having a DotNetVersion property